### PR TITLE
feat(mwpw-157946): UI-Tweaks for Newsroom

### PR DESCRIPTION
**Pull request by sheridansunier**
Resolves: [MWPW-157946](https://jira.corp.adobe.com/browse/MWPW-157946)
* adjust height of top filter text to match 'Filter:'

### CODE REVIEW

**Comment by sanrai** [comment]
@cmiqueo can you review this?

**Review by cmiqueo — APPROVED** [review]

File: less/components/consonant/filters/top/item.less [hunk 1]
```diff
@@ -55,8 +55,7 @@
         &-name {
             display: flex;
             align-items: center;
-
-            .no-margin;
+            margin: 2 0;
```
Comment (cmiqueo) [inline less/components/consonant/filters/top/item.less:58]: It should be: `margin: 2px 0;`

**Review by cmiqueo — CHANGES_REQUESTED** [review]
Please add a link to the jira ticket for easy cross-referencing :)

File: less/components/consonant/filters/top/item.less (hunk 1 above)
Comment (cmiqueo) [reply less/components/consonant/filters/top/item.less:58]: You should also add a _Resolves: <Link to jira ticket>_ section for easy cross reference :)

Comment (sheridansunier) [reply less/components/consonant/filters/top/item.less]: Both things done @cmiqueo :)

**Review by cmiqueo — APPROVED** [review]

**Pull request merged**